en/ros/ros1.md

```diff
@@ -1,14 +1,18 @@
-# ROS 1 with MAVROS
+# ROS 1 (Deprecated)
```

Comment:
FYI I decided that "with MAVROS" is not actually all that interesting any more. I guess it used to be because ROS1 had other transports for other systems, and may this was for those developers to understand why we don't use those. But anyhow, since we're deprecating this it really isn't relevant.

When ROS1 is really deprecated we should make decision about whether it is "supported" anymore.